runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -10093,8 +10093,7 @@ genHeapAlloc(TR::Node * node, TR::Instruction *& iCursor, bool isVariableLen, TR
    if (!comp->getOptions()->realTimeGC())
       {
       TR::Register *metaReg = cg->getMethodMetaDataRealRegister();
-
-      // bool sizeInReg = (isVariableLen || (allocSize > MAX_IMMEDIATE_VAL));
+      int32_t loops = allocSize / 256;
```

Comment:
Do we need this to be declared here? We should chose to declare and define variable closer to its use.